src/components/Notifications/NotificationFeed/NotificationFeed.js

```diff
@@ -106,7 +106,7 @@ class NotificationFeed extends React.Component {
     enableFilterBtn: true,
     enableFetchAllNotificationFeedAcrossProjects: false,
     selectedProjectId: 'all',
-    selectedProjectName: 'All Projects',
+    selectedProjectName: 'All',
```

Comment:
Since, you are checking this value later to equal "All" or not, making the "All" value a variable would be easier to maintain

Reply:
Good idea, thx